src/main/java/by/mifort/automation/hr/dev/util/MappingDtoComponentConverter.java

```diff
@@ -0,0 +1,34 @@
+package by.mifort.automation.hr.dev.util;
+
+import by.mifort.automation.hr.dev.dto.CommunicationHistoryDto;
+import by.mifort.automation.hr.dev.dto.KeywordDto;
+import by.mifort.automation.hr.dev.dto.CandidateDto;
+import by.mifort.automation.hr.dev.entity.CommunicationHistory;
+import by.mifort.automation.hr.dev.entity.Keyword;
+import by.mifort.automation.hr.dev.entity.Candidate;
+
+import java.util.List;
+
+/**
+ * Dto converter to entities
+ * @author yauheni_vozny
+ * @version 1.0
+ * */
+
+public interface MappingDtoComponentConverter {
```

Comment:
Can we use Spring Type Conversion?

Reply:
I discussed about this with Andrew and in conclusion, i will do it by myself